packages/components/atoms/f-button/package.json

```
@@ -45,15 +45,19 @@
     "extends @justeat/browserslist-config-fozzie"
   ],
   "peerDependencies": {
-    "@justeat/browserslist-config-fozzie": ">=1.1.1"
+    "@justeat/browserslist-config-fozzie": ">=1.1.1",
+    "vue": "2.x",
+    "vue-router": "^3.4.8"
```

Slightly different syntax require here.

```suggestion
    "vue-router": ">=3.4.8"
```